### ISSUE TITLE
Support for Python 3

### ISSUE THREAD

**Issue by kototama** [austinhyde/ansible-sshjail#22]
The software does not work with Python 3, a byte / string conversion error occurs.

**Comment by fdutheil** [comment]
A bit more details never hurts:
```The full traceback is:
Traceback (most recent call last):
  File "/usr/lib64/python3.6/site-packages/ansible/executor/task_executor.py", line 138, in run
    res = self._execute()
  File "/usr/lib64/python3.6/site-packages/ansible/executor/task_executor.py", line 566, in _execute
    result = self._handler.run(task_vars=variables)
  File "/usr/lib64/python3.6/site-packages/ansible/plugins/action/normal.py", line 46, in run
    result = merge_hash(result, self._execute_module(task_vars=task_vars, wrap_async=wrap_async))
  File "/usr/lib64/python3.6/site-packages/ansible/plugins/action/__init__.py", line 720, in _execute_module
    self._make_tmp_path()
  File "/usr/lib64/python3.6/site-packages/ansible/plugins/action/__init__.py", line 255, in _make_tmp_path
    tmpdir = self._remote_expand_user(remote_tmp, sudoable=False)
  File "/usr/lib64/python3.6/site-packages/ansible/plugins/action/__init__.py", line 590, in _remote_expand_user
    data = self._low_level_execute_command(cmd, sudoable=False)
  File "/usr/lib64/python3.6/site-packages/ansible/plugins/action/__init__.py", line 917, in _low_level_execute_command
    rc, stdout, stderr = self._connection.exec_command(cmd, in_data=in_data, sudoable=sudoable)
  File "/<blah>/ansible/connection_plugins/sshjail.py", line 325, in exec_command
    cmd = '%s %s %s %s' % (self.get_jail_connector(), self.get_jail_id(), cmd, '&& sleep 0')
  File "/<blah>/ansible/connection_plugins/sshjail.py", line 279, in get_jail_id
    self.match_jail()
  File "/<blah>/ansible/connection_plugins/sshjail.py", line 257, in match_jail
    lines = stdout.strip().split('\n')
TypeError: a bytes-like object is required, not 'str'
```
Anyway, non blocking issue here, as I can specify that I want ansible in python 2.7 flavor for now.

**Comment by austinhyde** [comment]
Sorry for letting this sit for so long...

I believe this is fixed by #25 which is now merged to master. Please let me know if that's not the case!
